srcpkgs/lz4/template

```diff
@@ -1,21 +1,22 @@
 # Template file for 'lz4'
 pkgname=lz4
-reverts="131_1 130_1 129_1 128_1 127_1 126_1 125_1 124_1 123_1 122_1"
-version=1.9.4
+version=1.10.0
 revision=1
-bootstrap=yes
-build_style=gnu-makefile
-make_check_target=test
+bootstrap="yes"
+build_style="gnu-makefile"
+make_check_target="test"
 checkdepends="python3"
-short_desc="LZ4 compression utilities"
-maintainer="Orphaned <orphan@voidlinux.org>"
-license="BSD-2-Clause, GPL-2.0-or-later"
+short_desc="Extremely fast compression algorithm"
+maintainer="Will Reed <wreed@disroot.org>"
```

Comment:
we'd prefer adoptions come from established contributors, especially those with prior experience working on the package
```suggestion
maintainer="Orphaned <orphan@voidlinux.org>"
```